netcore/NetCore.Session.IntegrationTests/SessionStateIntegrationTests.cs

```diff
@@ -0,0 +1,176 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+using System;
+using System.Text;
+using Xunit;
+using Apache.Geode.Client;
+using System.Linq;
+using Microsoft.Extensions.Caching.Distributed;
+using System.Threading.Tasks;
+
+namespace Apache.Geode.Session.IntegrationTests {
+  public class SessionStateIntegrationTests {
+    private static string _regionName = "exampleRegion";
+
+    [Fact]
+    public void SetGet() {
+      var ssCacheOptions = new GeodeSessionStateCacheOptions();
+      ssCacheOptions.Host = "localhost";
+      ssCacheOptions.Port = 10334;
+      ssCacheOptions.RegionName = "exampleRegion";
+
+      using var ssCache = new GeodeSessionStateCache(ssCacheOptions);
+
+      var options = new DistributedCacheEntryOptions();
+      DateTime localTime = DateTime.Now.AddDays(1);
+      DateTimeOffset dateAndOffset =
+          new DateTimeOffset(localTime, TimeZoneInfo.Local.GetUtcOffset(localTime));
+      options.AbsoluteExpiration = dateAndOffset;
+      var testValue = new byte[] { 1, 2, 3, 4, 5 };
+      ssCache.Set("testKey", testValue, options);
+      byte[] value = ssCache.Get("testKey");
```

Comment:
There are numerous places where we are not using `var`.

Reply:
Good catch.